src/util/text.js

```diff
@@ -127,3 +130,36 @@ export const parseEmailString = emails =>
   }))
 
 export const nounCount = (n, noun) => `${n} ${noun}${Number(n) !== 1 ? 's' : ''}`
+
+export const emailsFromCSVFile = file => {
+  const textType = /text.*/
+  return new Promise((resolve, reject) => {
+    if (!file.type.match(textType)) return reject('Not valid file format')
+    var reader = new FileReader()
```

Comment:
oh i see it's front-end only :+1: you can skip the warning by referring to it as window.FileReader